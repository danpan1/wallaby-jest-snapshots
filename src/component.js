import * as React from 'react';

export class TestComponent extends React.Component<TestProps, {}> {
  render() {
    return (
      <div className="root">
        <div className="string">String is {this.props.a}</div>
        <div className="number">Number is {this.props.b}</div>
      </div>
    );
  }
}