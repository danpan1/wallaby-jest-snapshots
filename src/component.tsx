import * as React from 'react';

export class TestComponent extends React.Component<any, any> {
  render() {
    return (
      <div className="root">
        <div className="string">String is {this.props.a}</div>
        <div className="number">Number is {this.props.b}</div>
      </div>
    );
  }
}