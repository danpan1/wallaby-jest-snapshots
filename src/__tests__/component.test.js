import {TestComponent} from '../component';
import * as React from 'react';
import {create} from 'react-test-renderer';

describe('component', () => {
  it('should be rendered correctly', () => {
    const tree = create(<TestComponent a="one string" b={1}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});