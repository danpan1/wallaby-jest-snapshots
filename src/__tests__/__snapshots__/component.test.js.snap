// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component should be rendered correctly 1`] = `
<div
  className="root"
>
  <div
    className="string"
  >
    String is 
    one string
  </div>
  <div
    className="number"
  >
    Number is 
    1
  </div>
</div>
`;
